experimental/LieAlgebras/src/root_systems.jl

```diff
@@ -0,0 +1,198 @@
+mutable struct RootSystem
+  roots::Vector{Vector{Int}}
+  simple_roots::Vector{Vector{Int}}
+  positive_roots::Vector{Vector{Int}}
+  root_system_type::Tuple{String, Int64}
+  @doc raw"""
+    RootSystem(S::String) -> RootSystem
+  Return the root system of type `S` where `S` is a string consisting out of
+  a letter `A`, `B`, `C`, `D`, `E`, `F`, `G` followed by an integer.
+  For the exceptional root system, the integers are fixed, e.g. `G2`, `F4`, `E6`, ...
+  """
+  function RootSystem(S::String)
+    # S is a string detailing the type of the indecomposable root system made out of a letter, e.g. "A", "B", "C",... and an integer for the number of simple roots
+    S = parse_root_string(S)
+    S1 = GAP.Obj(S[1])
+    RS = GAP.Globals.RootSystem(S1, S[2])
+    Ro_1 = GAP.Globals.PositiveRoots(RS)
+    Ro_2 = GAP.Globals.NegativeRoots(RS)
+    sR = GAP.Globals.SimpleSystem(RS)
+    sR = [[sR[i][j] for j=1:length(sR[i])] for i=1:length(sR)]
+    Ro1 = [[Ro_1[i][j] for j=1:length(Ro_1[i])] for i=1:length(Ro_1)]
```

Comment:
```suggestion
    Ro1 = Vector{Vector{Int}}(GAP.Globals.PositiveRoots(RS))
```
easier and faster. similar for the other lines